test/testutil/storage/base_fs_test.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_BASE_FS_TEST_HPP
+#define KAGOME_BASE_FS_TEST_HPP
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+
+#include "common/logger.hpp"
+
+// intentionally here, so users can use fs shortcut
```

Comment:
why not move it into fixture?

Reply:
namespace can not be renamed inside class